### PR TITLE
Re-enable a number of tests

### PR DESCRIPTION
**Pull request by vtnate**
#### Any background context you want to provide?
We were skipping a number of tests because they were failing. We now understand why some of them were failing, and have restored them to the CI test suite.

#### What does this PR accomplish?
- Pick up new docker image with OMv1.22.0 installed
- Mostly adds start, stop, and step-size to the simulations
- Improves logging inside the container
- Correctly place use_m_flow_in to be handled properly by 4G/5G systems
- Fix assert paths so tests don't fail
- Clarify skip rationale in the remaining skipped tests
#### How should this be manually tested?
- Run tests
    - The teaser tests take a loooooooooong time to simulate. I'm not sure what to do about that.
        - Maybe re-enable skipping those tests again while we figure it out?

### CODE REVIEW

File: geojson_modelica_translator/modelica/lib/runner/compile_fmu.most [hunk 1]
```diff
@@ -5,4 +5,4 @@ loadModel(Buildings);
 loadFile("{{ file_to_load }}");getErrorString();
 {% endif %}
 
-translateModelFMU({{ model_name }});getErrorString();
+buildModelFMU({{ model_name }});getErrorString();
```
Comment (vtnate) [inline geojson_modelica_translator/modelica/lib/runner/compile_fmu.most:8]: This raised a deprecation warning in OMv1.22, so I changed to the modern usage.

File: geojson_modelica_translator/modelica/model_connectors/templates/RunSpawnBuilding.mot [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+simulateModel(
+  "{{project_name}}.Loads.{{model_name}}",
+  stopTime=604800,
+  tolerance=1e-06,
+  resultFile="{{model_name}}"
+);
+createPlot(id=1, position={55, 50, 1783, 995}, y={ {% for zone in data['thermal_zones']%}"{{zone['modelica_object_name']}}.TAir", {% endfor %} {% raw %} "weaDat.weaBus.TDryBul", "datRea.y[1]", "datRea.y[2]", "datRea.y[3]", "datRea.y[4]", "datRea.y[5]", "datRea.y[6]", "datRea.y[7]"}, range={0.0, 620000.0, -30.0, 25.0}, grid=true, colors={{28,108,200}, {238,46,47}, {0,140,72}, {217,67,180}, {0,0,0}, {162,29,33}, {244,125,35}, {102,44,145}, {28,108,200}, {238,46,47}, {0,140,72}, {217,67,180}, {0,0,0}, {162,29,33}}, patterns={LinePattern.Solid, LinePattern.Solid, LinePattern.Solid, LinePattern.Solid,  LinePattern.Solid, LinePattern.Solid, LinePattern.Solid, LinePattern.Solid,  LinePattern.Dash, LinePattern.Dash, LinePattern.Dash, LinePattern.Dash,  LinePattern.Dash, LinePattern.Dash});{% endraw %}
```
Comment (nllong) [inline geojson_modelica_translator/modelica/model_connectors/templates/RunSpawnBuilding.mot:7]: we aren't really use these too often. I like the idea of having a user generated script to standard dymola plots, but that's for another day. Do these "createPlots" calls work in OM?

File: geojson_modelica_translator/modelica/modelica_runner.py [hunk 3]
```diff
@@ -150,7 +150,7 @@ def _copy_over_docker_resources(self, run_path: Path, filename: Union[str, Path,
         # new om_docker.sh file name
         new_om_docker = run_path / self.om_docker_path.name
         shutil.copyfile(self.om_docker_path, new_om_docker)
-        Path.chmod(new_om_docker, 0o775)
+        new_om_docker.chmod(0o775)
```
Comment (nllong) [inline geojson_modelica_translator/modelica/modelica_runner.py:153]: fancy....

File: tests/model_connectors/test_chp_system.py [hunk 4]
```diff
@@ -87,10 +87,13 @@ def test_build_chp_system(self):
         assert (root_path / 'DistrictEnergySystem.mo').exists()
 
     @pytest.mark.simulation
-    @pytest.mark.skip('OMC Failure: Initialization failed.')
+    @pytest.mark.skip('Compilation failed in OMv1.21. Works in Dymola')
```
Comment (nllong) [inline tests/model_connectors/test_chp_system.py:90]: `OM1.22`?

File: tests/model_connectors/test_time_series.py [hunk 5]
```diff
@@ -63,7 +63,7 @@ def test_build_model(self):
             self.assertTrue(os.path.exists(file), f"File does not exist: {file}")
 
     @pytest.mark.simulation
-    @pytest.mark.skip(reason="OMC Failure")
+    @pytest.mark.skip(reason="Because there is no district in this model, the GMT never instantiates `delTAirHea` and compilation fails.")
```
Comment (nllong) [inline tests/model_connectors/test_time_series.py:66]: 😳

**Review by nllong — APPROVED** [review]
wow, this is great. Seems like we are getting more coverage!

File: geojson_modelica_translator/model_connectors/load_connectors/templates/RunSpawnCouplingBuilding.most [hunk 6]
```diff
@@ -1,7 +1,6 @@
 old_hidden_avoid_double_computation=Hidden.AvoidDoubleComputation;
 Hidden.AvoidDoubleComputation=true;
 simulateModel("{{full_model_name}}",
-    method="cvode",
```
[the remark this answers is not in the document]
Comment (vtnate) [reply geojson_modelica_translator/model_connectors/load_connectors/templates/RunSpawnCouplingBuilding.most:4]: Yeah, OM defaults to DASSL

File: tests/model_connectors/test_chp_system.py (hunk 4 above)
Comment (vtnate) [reply tests/model_connectors/test_chp_system.py:90]: Ugh, old comment. Yes, that fails in 1.22 as well 😞

File: tests/model_connectors/test_time_series.py (hunk 5 above)
Comment (vtnate) [reply tests/model_connectors/test_time_series.py:66]: Yep 🤷

File: geojson_modelica_translator/modelica/model_connectors/templates/RunSpawnBuilding.mot (hunk 2 above)
Comment (vtnate) [reply geojson_modelica_translator/modelica/model_connectors/templates/RunSpawnBuilding.mot:7]: The spawn models don't work when I run them manually. I'll experiment with them in a new branch/PR.

File: geojson_modelica_translator/modelica/lib/runner/om.py [hunk 7]
```diff
@@ -145,14 +155,13 @@ def run_with_omc() -> bool:
         compile_fmu(model)
 
     if args.action == 'compile_and_run':
```
[the remark this answers is not in the document]
Comment (vtnate) [reply geojson_modelica_translator/modelica/lib/runner/om.py:157]: Working with Chris Mackey on a larger effort around the Docker container. That's a conversation for another day.

**Pull request merged**